lib/bigquery/job.js

```diff
@@ -156,35 +188,9 @@ nodeutil.inherits(Job, ServiceObject);
  * @example
  * job.cancel(function(err, apiResponse) {
  *   // Check to see if the job completes successfully.
- *   onJobComplete(function(err) {
- *     if (!err) {
- *       // Job cancelled successfully.
- *     }
- *   });
+ *   job.on('error', function(err) {});
+ *   job.on('complete', function(metadata) {});
```

Comment:
I think since it can take a long time to complete the operation/job, we have the separate type. All of the GCE operations are good examples of long-running operations (minutes). I think it's just a judgement call, however, if you think we should just wait to call the callback? // @jgeewax